tensorboard/webapp/notification_center/_data_source/notification_center_backend_types.ts

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+/**
+ * @fileoverview Types produced only by the backend data source.
+ *
+ * TODO(japie1235813): add api document
+ * Types defined in `plugins/notification_center/http_api.md`.
```

Comment:
FWIW, probably you can use the existing root-level `http_api.md` here that covers the other CorePlugin endpoints: https://github.com/tensorflow/tensorboard/blob/master/tensorboard/http_api.md

Reply:
Thanks for the comment. Remove this file and all references for now since it's not used yet.